ci/dev/prod_builders.json

```diff
@@ -25,6 +25,20 @@
          "repo":"engine"
       },
       {
+         "name":"Linux Web Framework tests",
+         "repo":"engine",
+         "enabled": true,
+         "run_if": [
+             "DEPS",
+             "lib/web_ui/**",
+             "web_sdk/**",
+             "e2etests/web/**",
+             "tools/**",
+             "ci/**",
+             "flutter_frontend_server/**"
```

Comment:
Filters are not needed in prod builders, we can remove them as well as the enabled property.

Reply:
Done.